src/components/user-profile-info/UserProfileInfo.tsx

```diff
@@ -57,17 +59,25 @@ const UserProfileInfo: FC<UserProfileInfoProps> = ({
     e.stopPropagation()
   }
 
+  const infoStyles = renderAdditionalInfo
+    ? spliceSx(sx.info, sx.interlocutorInfo)
+    : spliceSx(sx.info, sx.myInfo)
+
   return (
     <Box sx={spliceSx(styles.root, sx.root)}>
-      <Link onClick={handleLinkClick} to={userURL}>
-        <Avatar
-          src={photo && `${import.meta.env.VITE_APP_IMG_USER_URL}${photo}`}
-          sx={spliceSx(styles.avatar, sx.avatar)}
-        />
-      </Link>
-      <Box sx={spliceSx(styles.info, sx.info)}>
+      {renderAdditionalInfo && (
+        <Link onClick={handleLinkClick} to={userURL}>
+          <Avatar
+            src={photo && `${import.meta.env.VITE_APP_IMG_USER_URL}${photo}`}
+            sx={spliceSx(styles.avatar, sx.avatar)}
+          />
+        </Link>
+      )}
+      <Box sx={spliceSx(styles.info, infoStyles)}>
```

Comment:
Why you created this variable  
```
const infoStyles = renderAdditionalInfo
    ? spliceSx(sx.info, sx.interlocutorInfo)
    : spliceSx(sx.info, sx.myInfo)

```
and then use styles.info?
 Do it like this `spliceSx(styles.info, renderAdditionalInfo ? sx.interlocutorInfo : sx.myInfo)`

Reply:
> Why you created this variable
> 
> ```
> const infoStyles = renderAdditionalInfo
>     ? spliceSx(sx.info, sx.interlocutorInfo)
>     : spliceSx(sx.info, sx.myInfo)
> ```
> 
> and then use styles.info? Do it like this `spliceSx(styles.info, renderAdditionalInfo ? sx.interlocutorInfo : sx.myInfo)`

thank you for your advice, fixed it 👍